packages/create-contentful-app/template/src/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import { AppExtensionSDK, BaseExtensionSDK, init, locations } from 'contentful-ui-extensions-sdk';
+import '@contentful/forma-36-react-components/dist/styles.css';
+import '@contentful/forma-36-fcss/dist/styles.css';
+import './index.css';
+
+import Config from './components/ConfigScreen';
+import EntryEditor from './components/EntryEditor';
+import Page from './components/Page';
+import Sidebar from './components/Sidebar';
+import Field from './components/Field';
+import Dialog from './components/Dialog';
+
+init((sdk: BaseExtensionSDK) => {
+  const root = document.getElementById('root');
+
+  // All possible locations for your app
+  // Feel free to remove unused locations
+  // Dont forget to delete the file too :)
+  const ComponentLocationSettings = [
+    { location: locations.LOCATION_APP_CONFIG, component: <Config sdk={sdk as AppExtensionSDK} /> },
+    {
+      location: locations.LOCATION_ENTRY_FIELD,
+      component: <Field />
+    },
+    {
+      location: locations.LOCATION_ENTRY_EDITOR,
+      component: <EntryEditor />
+    },
+    { location: locations.LOCATION_DIALOG, component: <Dialog /> },
+    {
+      location: locations.LOCATION_ENTRY_SIDEBAR,
+      component: <Sidebar />
+    },
+    { location: locations.LOCATION_PAGE, component: <Page /> }
```

Comment:
Should all of these components above get passed the `sdk` with their corresponding interfaces?

Reply:
I thought of this but we're not really using the sdk in any of these. I opted to leave config one with sdk and remove from the others. Do you see anyone not getting that sdk can be passed in as a prop? Or in other words, is there a benefit of passing sdk into these?

Reply:
I think from an educational perspective it would be nice to pass them in even if they don't do anything because it will also expose the use of the different TS interfaces that have to be declared along with it.

For example, devs will see there is not only an `AppExtensionSDK` but also a `PageExtensionSdk` interface and more.